api/instance/v1/server_utils.go

```diff
@@ -51,3 +52,51 @@ func (s *API) WaitForServer(req *WaitForServerRequest) (*Server, error) {
 
 	return server.(*Server), err
 }
+
+// DoActionAndWaitRequest is used by DoActionAndWait method
+type DoActionAndWaitRequest struct {
+	ServerID string
+	Zone     utils.Zone
+	Action   ServerAction
+	Timeout  time.Duration
```

Comment:
Add comment with default value  and add a default value

Reply:
I don't like to rely on someone else default value  (in this case `waitForServer`) in a method